### ISSUE TITLE
Cryptic error when faced with malformed coadd input table

### ISSUE THREAD

**Issue by erykoff** [LSSTDESC/supreme#26]
@jchiang87 reports the following error on what turns out to be a bad patch:
```
    ccd_poly = ccd.getValidPolygon().convexHull()
AttributeError: 'NoneType' object has no attribute 'convexHull'
```
A check should be added to see if `ccd.getValidPolygon()` actually returns a valid polygon, and if not raise a more informative error.

**Comment by erykoff** [comment]
Turns out this is work-around-able: https://github.com/lsst/pipe_tasks/blob/master/python/lsst/pipe/tasks/assembleCoadd.py#L1022

**Comment by erykoff** [comment]
Closed by #28
